### PR TITLE
AR-84: Change background colour for elements on opinion/comment articles

### PR DESCRIPTION
**Pull request by frankie297**
## Why are you doing this?
Some of the contrast of background elements were not clear enough on opinion/comment articles, this PR changes the background colour from `neutral.97` to `neutral.86`

This is for:

- header images
- header videos
- body images
- body videos
- adverts
- tags
- rich links

Paired with @dskamiotis 

## Screenshots
Header
| Before | After |
| --- | --- |
|  |  |

Body
| Before | After (with Advert too) |
| --- | --- |
|  |  |

Tags and Richlink
| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

**Comment by frankie297** [comment]
@JamieB-gu @webb04 There are a couple of random unit tests to make the CI happy, but if you would rather they were more relevant to the PR then I might need some help 😄

File: src/components/headerVideo.tsx [hunk 1]
```diff
@@ -18,13 +18,24 @@ const marginAuto = `
...
+const backgroundColour = (format: Format): string => {
+    switch (format.design) {
+        case Design.Media:
+            return neutral[20];
+        case Design.Comment:
+            return neutral[86];
+        default:
+            return neutral[97];
+    }
+};
```
Comment (JamieB-gu) [inline src/components/headerVideo.tsx:30]: Nice 👍

File: src/components/shared/tags.test.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { configure, shallow, ShallowWrapper } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import Tags from 'components/shared/tags';
 import { SerializedStyles } from '@emotion/core';
+import { Design, Display, Format, Pillar } from "../../format";
```
Comment (JamieB-gu) [inline src/components/shared/tags.test.tsx:6]:
```suggestion
import { Design, Display, Format, Pillar } from '@guardian/types/Format';
```

File: src/renderer.test.ts [hunk 3]
```diff
@@ -119,6 +119,22 @@ const atomElement = (): BodyElement =>
         js: some("console.log('init')"),
     })
 
+const liveEventElement = (): BodyElement =>
+    ({
+        kind:ElementKind.LiveEvent,
```
Comment (JamieB-gu) [inline src/renderer.test.ts:124]:
```suggestion
        kind: ElementKind.LiveEvent,
```

File: src/renderer.test.ts [hunk 4]
```diff
@@ -119,6 +119,22 @@ const atomElement = (): BodyElement =>
...
+        linkText: "this links to a live event",
+        url: "https://gu.com/liveevent"
+    })
+
+const explainerElement = (): BodyElement =>
+    ({
+        kind:ElementKind.ExplainerAtom,
```
Comment (JamieB-gu) [inline src/renderer.test.ts:131]:
```suggestion
        kind: ElementKind.ExplainerAtom,
```

**Pull request merged**